tests/filecheck/dialects/print/print_basics.mlir

```diff
@@ -10,6 +10,8 @@ builtin.module {
 
     printf.print_format "{}", %144 : i32
     printf.print_format "{}", %144 : i32 {unit}
+    "printf.print_char"(%12) : (i32) -> ()
```

Comment:
hmm, what size should char be? feels like `u8` to me, but I'm really not sure

Reply:
I've used signless 8-bit types in the latest update